components/mixins/profile/profile-internet-subscription.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/profile/profile-internet-subscription",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Internet Subscription",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": { "industry": ["telecom"] },
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/mixins/profile/profile-telecom-subscription"
```

Comment:
I am guessing this is still a mixin which extends profile class and needs to have all the properties from "profile/profile-telecom-subscription". If my assumption is right, please change the "meta:intendedToExtend" to profile class and add a "meta:extends" to add the "profile/profile-telecom-subscription" schema. This applies to all the subscription schemas you are creating in this PR.